client/src/app/pages/assessment/import-questionnaire-form/import-questionnaire-form.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import axios, { AxiosResponse } from "axios";
+import { useTranslation } from "react-i18next";
+import * as yup from "yup";
+
+import {
+  ActionGroup,
+  Button,
+  ButtonVariant,
+  Checkbox,
+  FileUpload,
+  Form,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from "@patternfly/react-core";
+
+import { UPLOAD_FILE } from "@app/api/rest";
+import { getAxiosErrorMessage } from "@app/utils/utils";
+import { NotificationsContext } from "@app/components/NotificationsContext";
+import { HookFormPFGroupController } from "@app/components/HookFormPFFields";
+import { useForm } from "react-hook-form";
+import { FileLoadError, IReadFile } from "@app/api/models";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useCreateFileMutation } from "@app/queries/targets";
+
+export interface ImportQuestionnaireFormProps {
+  onSaved: (response: AxiosResponse) => void;
+}
+export interface ImportQuestionnaireFormValues {
+  yamlFile: IReadFile;
+}
+
+export const yamlFileSchema: yup.SchemaOf<IReadFile> = yup.object({
+  fileName: yup.string().required(),
+  fullFile: yup.mixed<File>(),
+  loadError: yup.mixed<FileLoadError>(),
+  loadPercentage: yup.number(),
+  loadResult: yup.mixed<"danger" | "success" | undefined>(),
+  data: yup.string(),
+  responseID: yup.number(),
+});
+
+export const ImportQuestionnaireForm: React.FC<
+  ImportQuestionnaireFormProps
+> = ({ onSaved }) => {
+  const { t } = useTranslation();
+  const { pushNotification } = React.useContext(NotificationsContext);
+
+  const [filename, setFilename] = React.useState<string>();
```

Comment:
React. is not needed as you already import useState.
I also think it would be nice to remove all React. occurences from this file and import once at the top